draft-mdt-quic-explicit-measurements.md

```diff
@@ -322,19 +340,20 @@ upstream paths with different latency characteristics.
 
 The use of the loss bits is negotiated using a transport parameter:
 
-loss_bits (0x1057):
+enable_network_troubleshooting (0x1057):
```

Comment:
I like this change a lot.  It is very generic and purposeful.  As discussed below, however, we _may_ still want to maintain an ability to say: "I will allow you to use reserved bits for this purpose, but I will not send my own loss information to you."

Reply:
+1: this asymmetric (but now bilaterally agreed) mode is what makes (e.g.) the QL deployable at large scale: the server asks the permission to send, many clients grant it, but they don't have any need to implement more than the TP-accept code.